File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacBusiness/PaisesBO.cs
using SoftPac.Model;
using SoftPac.Persistance.DAO;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SoftPac.Business
{
    public class PaisesBO
    {
        PaisesDAO paisesDAO;

        public PaisesBO()
        {
            this.paisesDAO = new PaisesDAOImpl();
        }
        public BindingList<PaisesDAO> ListarTodos()
        {
            return (BindingList<PaisesDAO>)this.paisesDAO.ListarTodos();
        }
    }
}

File: Backend_Java/SoftPacBusiness/src/main/java/pe/edu/pucp/softpac/bo/PaisesBO.java

package pe.edu.pucp.softpac.bo;

import java.util.ArrayList;
import pe.edu.pucp.softpac.dao.PaisesDAO;
import pe.edu.pucp.softpac.daoImpl.PaisesDAOImpl;
import pe.edu.pucp.softpac.model.PaisesDTO;

public class PaisesBO {
    
    private PaisesDAO paisDAO;
    
    public PaisesBO(){
        paisDAO = new PaisesDAOImpl();
    }
    
    public ArrayList<PaisesDTO> listarTodos(){
        return (ArrayList<PaisesDTO>) this.paisDAO.listarTodos();
    }
    
    public PaisesDTO obtenerPorId(Integer pais_id){
        return this.paisDAO.obtenerPorId(pais_id);
    }
    
}
